db/migrate/20241227165942_add_unique_index_to_languages.rb

```diff
@@ -0,0 +1,10 @@
+class AddUniqueIndexToLanguages < ActiveRecord::Migration[6.1]
```

Comment:
It looks like this migration was generated with Rails 6.1 rather than Rails 7. (Here's one from a recent PR that uses Rails 7: https://github.com/otwcode/otwarchive/pull/4999/files#diff-c8c18752ab7910e1481946e61b4e79c6ea31a75cf374308db8ced589f489cdb4R3). Would you mind updating this so it also uses Rails 7? I think that should be as simple as changing this to the following
```suggestion
class AddUniqueIndexToLanguages < ActiveRecord::Migration[7.0]
```